website/static/js/registrationEditorExtensions.js

```diff
@@ -103,7 +103,9 @@ var osfUploader = function(element, valueAccessor, allBindings, viewModel, bindi
                 limitContents(item);
 
                 if (viewModel.value()) {
-                    if (item.data.path === viewModel.value()) {
+                    if (item.data.name === viewModel.value()) {
+                        item.css = 'fangorn-selected';
+                    } else if ( viewModel.value() === 'No file selected' && item.data.name === 'OSF Storage') {
```

Comment:
This looks brittle--it will break if this text changes. Is there a more stable value that can checked against?